cli/src/main/kotlin/OrtMain.kt

```diff
@@ -87,7 +87,7 @@ class OrtMain : CliktCommand(name = ORT_NAME, epilog = "* denotes required optio
     private val configArguments by option(
         "-P",
         help = "Override a key-value pair in the configuration file. For example: " +
-                "-P scanner.postgresStorage.schema=testSchema"
+                "-P ort.scanner.storages.postgresStorage.schema=testSchema"
```

Comment:
Commit message *title* Please use

    OrtMain: Fix a property reference in the help text of the -P option

(You don't need to say it's wrong; that's implied by you saying to fix it.)

Reply:
Hmm, I remember we discussed the wrapping in the top-level "ort" namespace as part of your PR, and I thought you were adding code to ensure the behavior is the same as before and "ort" does not need to specified here. But maybe I'm also mixing things up, and "ort" was always required here and the example was always wrong. Anyway, @mnonnenmacher, I thought you're using this mechanism somewhere. And did the behavior change recently for you?

Reply:
It seems to me that the example was always wrong. The old implementation of OrtConfiguration had the line:
`val argsConfig = ConfigFactory.parseMap(args, "Command line").withOnlyPath("ort")`

So only properties starting with the prefix 'ort' have been taken into account. This is still the case; so there has not been a change of the behavior.

Reply:
Thanks for looking this up to clarify!